packages/dynamite/lib/src/openapi_builder.dart

```diff
@@ -1862,3 +1877,16 @@ TypeResult resolveType(
 
   throw Exception('Can not convert OpenAPI type "${schema.toJson()}" to a Dart type');
 }
+
+// ignore: avoid_positional_boolean_parameters
+int sortRequired(final bool a, final bool b) {
```

Comment:
This took me a bit to understand.

Maybe going forward we should add doc comments to those stuff so others can understand easier.

Reply:
Yeah, I agree. Dart sort methods are pretty weird to use imo.

Reply:
We don't need to explain sort methods but mentioning it is one is enough.

My head expanded the `sortRequired` into `isSortRequired` and I then expected it have a boolean return type.

Just mentioning it is a sort method for required params would have be it.

Reply:
Is `sortRequiredElements` good? It sorts both parameters and properties, so it has to be agnostic.